awx/ui_next/src/screens/Job/JobDetail/JobDetail.test.jsx

```diff
@@ -57,4 +61,57 @@ describe('<JobDetail />', () => {
       job.summary_fields.credentials[0]
     );
   });
+  test('should properly delete job', () => {
+    job = {
+      name: 'Rage',
+      id: 1,
+      type: 'job',
+      summary_fields: {
+        job_template: { name: 'Spud' },
+      },
+    };
+    const wrapper = mountWithContexts(<JobDetail job={job} />);
+    wrapper
+      .find('button')
+      .at(0)
+      .invoke('onClick')();
+    const modal = wrapper.find('Modal');
+    expect(modal.length).toBe(1);
+    modal.find('button[aria-label="delete"]').invoke('onClick')();
```

Comment:
I _think_ we need to `.simulate` the click instead of using `.invoke` to avoid the `.act()` React warnings.

Reply:
It looks like React addresses this very issue in v16.9.0.
https://reactjs.org/blog/2019/08/08/react-v16.9.0.html#react-test-utilities-and-test-renderer